src/ontology/cl-edit.owl

```diff
@@ -32081,6 +32083,27 @@ AnnotationAssertion(rdfs:label obo:CL_4042011 "subpial interlaminar astrocyte"@e
 SubClassOf(obo:CL_4042011 obo:CL_4042009)
 SubClassOf(obo:CL_4042011 ObjectSomeValuesFrom(obo:RO_0002170 obo:UBERON_0002361))
 
+# Class: obo:CL_4042012 (L2 marginal neuron)
+
+AnnotationAssertion(obo:IAO_0000028 obo:CL_4042012 "L2MN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29311847") Annotation(oboInOwl:hasDbXref "PMID:32108571") obo:IAO_0000115 obo:CL_4042012 "An L2 intratelencephalic projecting glutamatergic neuron with a soma on the L1-L2 border. This neuron type has small apical dendrites projecting to L1.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042012 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042012 "2024-04-23T09:31:39Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29311847") Annotation(oboInOwl:hasDbXref "PMID:32108571") oboInOwl:hasExactSynonym obo:CL_4042012 "L2MN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32108571") oboInOwl:hasExactSynonym obo:CL_4042012 "layer 2 marginal neuron")
+AnnotationAssertion(rdfs:label obo:CL_4042012 "L2 marginal neuron"@en)
+SubClassOf(obo:CL_4042012 obo:CL_4023040)
+
+# Class: obo:CL_4042013 (Lamp5 Lhx6 neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34004146") Annotation(oboInOwl:hasDbXref "PMID:36007006") Annotation(oboInOwl:hasDbXref "PMID:37824655") Annotation(oboInOwl:hasDbXref "PMID:37824669") obo:IAO_0000115 obo:CL_4042013 "A transcriptomically distinct lamp5 GABAergic cortical interneuron located in the cerebral cortex that expresses Lamp5 and Lhx6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_cluster', Lamp5 Lhx6.")
```

Comment:
Since this is a new way for defining cells, it should be included to the editor SOP. IMO I would not include this in the definition, but I believe it has been discussed elsewhere to adopt this style for cellxgene new terms.

Reply:
Perhaps it is not conventional.
@dosumis we should probably discuss this in the next CL internal meeting.